doc/source/remote_reading.rst

```diff
@@ -0,0 +1,81 @@
+====================
+Reading remote files
```

Comment:
For consistency, should this page contain the FSFile examples too?

Reply:
Yes, I'll add them.